settings.yaml.template

```diff
@@ -28,7 +28,7 @@ PROVIDERS:
   EC2:
     USERNAME:
     PASSWORD:
-    REGION:
+    REGION: []
```

Comment:
@shubhamsg199 Lets move it to regions since its a list of regions be its with one or many regions :)

Also add a comment above as an example of how the regions could be defined here. Mentions some regions name there.